doc/source/rayserve/tensorflow-tutorial.rst

```diff
@@ -1,16 +1,20 @@
-Tensorflow Guide
-================
+Keras and Tensorflow Tutorial
+=============================
 
-In this guide, we will train and deploy a simple Tensorflow neural net. 
+In this guide, we will train and deploy a simple Tensorflow neural net.
 In particular, we show:
 
 - How to load the model from file system in your RayServe definition
 - How to parse the JSON request and evaluated in Tensorflow
 
-This tutorial requires Tensorflow 2. Please make sure you have Tensorflow 2 installed.
+Please see the :ref:`overview <rayserve-overview>` to learn more general information about RayServe.
+
+RayServe supports arbitrary frameworks. However, for this tutorial, we use
```

Comment:
```suggestion
RayServe makes it easy to deploy models from all popular frameworks. However, for this tutorial, we use
```

Reply:
+1, this is always a great idea because it allows people to learn more. @richardliaw and I did a bunch of this for Tune in his last PR: https://github.com/ray-project/ray/pull/8201

Linking is easy to do and provides a great way for other people to "self-discover" what they want to use.